cfme/tests/test_rest.py

```diff
@@ -1306,3 +1307,42 @@ def test_release_server_info(appliance):
         appliance.rest_api.server_info["release"]
         == appliance.ssh_client.run_command("cd /var/www/miq/vmdb; cat RELEASE").output
     )
+
+
+@pytest.mark.customer_scenario
+@pytest.mark.meta(automates=[1661445])
+@pytest.mark.tier(1)
+def test_authorization_header_sql_response(appliance):
+    """
+    Bugzilla:
+        1661445
+        1686021
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Rest
+        initialEstimate: 1/4h
+        testSteps:
+            1. GET /api/auth?requester_type=ui HEADERS: {"Authorization": "Basic testing"}
+                Perform this GET request with requests.get()
+        expectedResults:
+            1. There should be no sql statement in the response.
+                Expected Response:
+                {
+                "error": {
+                    "kind": "unauthorized",
+                    "message": ("PG::CharacterNotInRepertoire: ERROR:"
+                                "  invalid byte sequence for encoding \"UTF8\": 0xb5\n:"),
+                    "klass": "Api::AuthenticationError"
+                    }
+                }
+    """
+    appliance.rest_api._session.headers = {"Authorization": "Basic testing"}
```

Comment:
@valaparthvi you are overtiring attributes of `ManageIQClient` instance.
I think this is special case where we need different header than `{'Content-Type': 'application/json; charset=utf-8'}`. I would instantiate separate instance of `MangeIQClient` than this workaround. what you say?

Reply:
Great suggestion @digitronik. Alternatively this needs to be reset on the session with a request.addfinalizer call.

Reply:
Good catch @digitronik @mshriver ! Changed the test to use a seperate instance.